mlrun/runtimes/nuclio/function.py

```diff
@@ -510,6 +510,7 @@ def add_v3io_stream_trigger(
                 **kwargs,
             ),
         )
+        logger.warning(f"Setting function replicas to {shards}")
```

Comment:
I think you should check if there is some non-zero value first in the min/max replicas, and warn that is being **overridden**.

Otherwise it doesn't matter much.

Reply:
@TomerShor agree, fixed